### DISCUSSION TITLE
[Vue] Switch component suddenly stopped working

### DISCUSSION THREAD

**Discussion by giusepetroso** [tailwindlabs/headlessui#1637] [Help]
Hi guys,

maybe it's just me messed up something but in a vue/vite app Switch component suddenly stopped working, throwing this error in console:
`Uncaught TypeError: Cannot read properties of undefined (reading 'switchRef')`

I just commented out everything else from my main app component and pasted just the example code of the Switch to understand what was happening, but I can't find a way to get it work
![image](https://user-images.githubusercontent.com/17656277/177002602-b539a3cf-01e9-4cfe-aea4-8bda4a76792a.png)
![image](https://user-images.githubusercontent.com/17656277/177002642-004404ea-0d2b-4fa8-bed9-5008cf50e65d.png)

Version:
@headlessui/vue: "^1.6.5"

I can't figure out what is wrong, hope you can help me.
If you need some additional info let me know.

Thanks, 
Giuse

**Comment by banocean** [comment]
Same problem (https://github.com/tailwindlabs/headlessui/issues/1827)
